etc/inc/disable-programs.inc

```diff
@@ -94,6 +94,7 @@ blacklist ${HOME}/.config/Gitter
 blacklist ${HOME}/.config/Google
 blacklist ${HOME}/.config/Google Play Music Desktop Player
 blacklist ${HOME}/.config/Gpredict
+blacklist ${HOME}/.config/homebank
```

Comment:
disable-programs is sorted alphabetically (as you saw) and case sensitive [with ~/.cache/* in a extra block]. So move it down (239:hexchat).

Reply:
Thanks, I thought they were sorted by "updated".